src/propfind.cpp

```diff
@@ -75,7 +75,37 @@ static bool readPrivilegeSet(QXmlStreamReader *reader, bool *readOnly)
     return false;
 }
 
-static bool readCalendarProp(QXmlStreamReader *reader, bool *isCalendar, QString *label, QString *color, QString *userPrincipal, bool *readOnly)
+static bool readComponentSet(QXmlStreamReader *reader,
+                             bool *allowEvents, bool *allowTodos, bool *allowJournals)
+{
+    /* e.g.:
+                    <C:supported-calendar-component-set>
+                        <C:comp name="VEVENT" />
+                    </C:supported-calendar-component-set>
```

Comment:
Cosmetics: could indent slightly less?

Reply:
Sure, I copy pasted for some other palces where the indentation is large also, but no need to keep such space.